services/app-api/handlers/reports/create.ts

```diff
@@ -232,8 +239,7 @@ export const createReport = handler(
         ),
         targetPopulations: "Older adults, PD, MH/SUD",
         reportYear,
-        dueDate: currentDate,
-        reportPeriod: calculatePeriod(currentDate, workPlanMetadata),
+        dueDate: calculateDueDate(reportYear, reportPeriod, reportType),
         associatedWorkPlan: workPlanMetadata?.id,
       },
```

Comment:
did you mean to remove `reportPeriod`?

Reply:
whoops! I did not. Good catch! I'll add it back in